source/hardware/controllers/numark_dj2go2_touch.rst

```diff
@@ -0,0 +1,32 @@
+.. _numark-dj2go2-touch:
+
+Numark DJ2GO2 Touch
+===================
+
+.. sectionauthor::
+   tandy1000 <https://github.com/tandy-1000>
+
+The Numark DJ2GO2 Touch is a USB :term:`MIDI` controller with an integrated audio interface.
+It is the successor to the DJ2GO2, and features touch capable wheels.
+
+-  `Manufacturer’s product page <https://www.numark.com/product/dj2go2-touch>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/numark-dj2go2-touch>`__
+-  `Mapping Forum Thread <https://mixxx.discourse.group/t/a-complete-numark-dj2go2-touch-mapping/22616>`__
+
+.. versionadded:: 2.3.1
+
+Controller Mapping
+------------------
+
+Everything in this mapping is standard except:
```

Comment:
```suggestion
Everything in this mapping works as labeled on the controller except:
```

Reply:
Could you please add a list of controls like Roland DJ-505 mapping does?

Some users may never use the controller with another DJ software. And "everything is mapped to standard" is ambiguous. For example, what does the Sync button do? Does it do a one-shot beatsync? Or does it toggle sync lock?